apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/success/page.tsx

```diff
@@ -121,12 +120,12 @@ export default async function CheckoutSuccess({
             )}
           </div>
           <div className="flex items-center justify-between text-grey-400">
-            <PriceInterval
-              l10n={l10n}
-              amount={cart.latestInvoicePreview?.totalAmount}
-              currency={cart.latestInvoicePreview?.currency}
-              interval={cart.interval}
-            />
+            {formatPriceAmount(
```

Comment:
This string needs to be localized, which `formatPriceAmount` doesn't currently do.

For example, an issue might come up with a string like `$5.00 + 0.45 tax` not being translated. (Eg. For German it would be something like `$5.00 + 0.45 Steuern`)

Reply:
Ah I see we don't currently show taxes in this field.

It did get me wondering though, are we localizing the currency field correctly? I believe in German it's value first then currency, so it should be `5,45 $` instead of `$5.00`.

Could we get that fixed as part of this PR if possible?